app/models/group.rb

```diff
@@ -277,6 +280,10 @@ def remove(user)
     self.group_users.where(user: user).each(&:destroy)
   end
 
+  def appoint_manager(user)
```

Comment:
I put that in so there is a place for the controller to invoke once there is a UI for this (I'm working on it, slowly). Not sure where the best separator is between back-end and front-end stuff, and I didn't want to touch the front-end with this patch.